polaris/ocean/tasks/baroclinic_channel/init.py

```diff
@@ -162,8 +163,30 @@ def run(self):
         write_netcdf(ds, 'initial_state.nc')
 
         cell_mask = ds.maxLevelCell >= 1
-        plot_horiz_field(ds, ds_mesh, 'temperature',
-                         'initial_temperature.png', cell_mask=cell_mask)
+
         plot_horiz_field(ds, ds_mesh, 'normalVelocity',
                          'initial_normal_velocity.png', cmap='cmo.balance',
                          show_patch_edges=True, cell_mask=cell_mask)
+
+        x_mid = 0.5 * (x_min + x_max)
```

Comment:
Optional: use the median value of xCell so that the transect goes through the cell centers

Reply:
I noticed that you had done that in your test case.  It's a smart idea.  I'll do it here, too.

Reply:
Done!